### DISCUSSION TITLE
ipv6支持吗？

### DISCUSSION THREAD

**Discussion by suyuanhua** [vxiaov/vClash#24] [Q&A] [answered]
ipv6支持吗？配置示例里好像有，容我试试支持IPv6🙏
感谢作者！

**Comment by Awkee** [comment, marked as answer]
> ipv6支持吗？配置示例里好像有，容我试试支持IPv6pray 感谢作者！

选项里有支持IPv6功能，看能不能满足你使用要求，可以保证正常访问IPv6网站，至于能不能走有IPv6地址的代理节点我没实际验证过。
